unicodetools/data/ucd/dev/IndicPositionalCategory.txt

```diff
@@ -1,11 +1,11 @@
-# IndicPositionalCategory-15.1.0.txt
-# Date: 2023-01-05
+# IndicPositionalCategory-16.0.0.txt
+# Date: 2023-10-02, 22:58:33 GMT
 # © 2023 Unicode®, Inc.
 # Unicode and the Unicode Logo are registered trademarks of Unicode, Inc. in the U.S. and other countries.
 # For terms of use, see https://www.unicode.org/terms_of_use.html
 #
-# For documentation, see UAX #44: Unicode Character Database,
-# at https://www.unicode.org/reports/tr44/
+# Unicode Character Database
+#   For documentation, see https://www.unicode.org/reports/tr44/
```

Comment:
question only: Better consistent with other files but not with Roozbeh's files here? Will we see flip-flopping between Roozbeh sending updates and us regenerating these files?

Reply:
We will see flip-flopping unless or until Roozbeh updates this part of his template, something similar (though it was just the indentation) happened with Ken’s LineBreak.txt.

This header is generated in a different place, so it would be mildly annoying to pass a Roozbeh flag around to customize it.